src/ctypes/ctypes_memory_stubs.ml

```diff
@@ -48,4 +48,4 @@ external string_of_array : _ Fat.t -> len:int -> string
 
 (* Do nothing, concealing from the optimizer that nothing is being done. *)
 external use_value : 'a -> unit
-  = "ctypes_use"
```

Comment:
Isn’t this deprecated in favour of [@@noalloc]?

Reply:
ctypes is currently compatible with OCaml versions back to 4.01, which doesn't support attributes.

Reply:
#621 switches to attributes, now that we've dropped OCaml 4.01 support.